openmetadata-ui/src/main/resources/ui/src/components/common/table-data-card/TableDataCard.tsx

```diff
@@ -130,6 +132,14 @@ const TableDataCard: FunctionComponent<Props> = ({
               {stringToHTML(name)}
             </button>
           </h6>
+          {deleted && (
+            <>
+              <div className="tw-rounded tw-bg-error-lite tw-text-error tw-text-xs tw-font-medium tw-h-5 tw-px-1.5 tw-py-0.5 tw-ml-2">
```

Comment:
We should make it a separate component if we are using it in more than one place

Reply:
UI might be changed so we can do it once it's final